### ISSUE TITLE
Custom dataset evaluation

### ISSUE THREAD

**Issue by messimm** [mhamilton723/STEGO#21]
Hi! Thank you for your code!

Can you explain process of evaluation for custom dataset in Readme? 
What is picie? dark_mode? How to launch evaluation without errors?

Thank you for cooperation!

**Comment by mhamilton723** [comment]
Hey @messimm, thanks for reaching out. PiCIE was the baseline we compared against so setting that to True runs a pre-trained PiCIE model through the same evaluation. Note that for your use case you can turn that to False because you dont have pre-trained PiCIE weights. 

Dark mode is just whether to plot the plots with a black background for making pretty slides. 

What errors are you encountering specifically?

**Comment by Eric-L-Manibardo** [comment]
Hello. I am currently trying to evaluate STEGO using the cityscapes_vit_base_1.ckpt model over a custom dataset. I would like to ask for the steps toward setting the dataset. I have 3 folders with train, val, and test data. Since the model is already trained, only test data should be used. Therefore it is okey for setting a folder at e.g. 'datadrive/pytorch-data/test'? In this folder there are several images in PNG format.

The final output that I want is all these images from the 'test' folder segmented by STEGO. Please let me know if I am on the way or if perhaps I misunderstood how it works since I am new to the Computer Vision realm. Thanks in advance!

**Comment by innat-asj** [comment]
@mhamilton723 

- Why is it computing like this? https://github.com/mhamilton723/STEGO/blob/eb4d6b521740bd4265681b353547f2ffca65d673/src/eval_segmentation.py#L103-L113
- Is it possible to evaluate data without labels? (maybe k-nn classifier!)
